app/assets/javascripts/proto_plugin/main.js

```diff
@@ -6,13 +6,25 @@
 // For this example plugin the file is included into every page through the
 // :header_tags view hook (see `lib/open_project/proto_plugin/hooks.rb`).
 
-jQuery(document).ready(function() {
-  jQuery("#logo").hover(
-    function enter() {
-      jQuery(this).attr("style", "border: 3px solid red;");
-    },
-    function leave() {
-      jQuery(this).removeAttr("style");
-    }
-  );
-});
+(function($) {
+    $(document).ready(function() {
+
+      // OpenProject logo gets thick red border on mouse hover.
+      $("#logo").hover(
+        function enter() {
+          $(this).attr("style", "border: 3px solid red;");
+        },
+        function leave() {
+          $(this).removeAttr("style");
+        }
+      );
+
+      // Widget box emphasized by giving it a nice red border.
+      $('#proto-plugin-block').parent().addClass('proto-plugin-widget-box');
+
+      console.log('Proto plugin OK');
+
+      // TODO
+      //console.log(I18n.t('proto_plugin_name') + ' OK');
```

Comment:
You can add a `config/locales/js-en.yml` file to add locale definitions for the frontend [`I18n.js`](https://github.com/fnando/i18n-js). It will automatically be picked up by the main application.

As a convention (cf., a different plugin), we always use `js` as the main namespace in each `js-en.yml` file.
So the contents of `js-en.yml` would become:

```
en:
  js:
    proto_plugin_name: 'Kitten plugin'
    
```